lang/hr.js

```diff
@@ -0,0 +1,217 @@
+        "find-more": "Saznaj više"
+      },
+      "troubleshoot": "Rješavanje problema",
+      "close": "Zatvori",
+      "request-repl": "Pošalji CTRL-C za REPL",
+      "request-serial": "Pošalji CTRL-D za krenuti ispočetka",
+      "flashing-text": "\"Flashanje\" micro:bita (prijenos datoteke)",
+      "download": "Preuzmi .hex"
+    },
+    "load": {
+      "show-files": "Pokaži datoteke",
+      "load-title": "Učitaj",
+      "instructions": "Povuci i spusti .hex ili .py datoteku ovamo da se otvori.",
+      "submit": "Učitaj",
+      "save-title": "Spremi",
+      "save-hex": "Preuzmi .hex datoteku projekta",
+      "save-py": "Preuzmi Pythonov programski kôd",
+      "fs-title": "Datoteke",
+      "toggle-file": "Ili potraži datoteku pomoću pretraživača.",
+      "fs-add-file": "Dodaj datoteku",
+      "hide-files": "Sakrij datoteke",
+      "td-filename": "Ime datoteke",
+      "td-size": "Veličina",
+      "fs-space-free": "slobodno",
+      "remove-but": "Izbaci",
+      "save-but": "Spremi",
+      "files-title": "Datoteke projekta",
+      "help-button": "Pomoć za datoteke",
+      "file-help-text": "U području projektnih datoteka vidi se koje su sve datoteke uključene u tvoj program i tu se mogu dodavati ili uklanjati vanjski Pythonovi moduli i druge datoteke. Saznaj više u ",
+      "help-link": "Dokumentacija za pomoć s Pythonovim uređivačem",
+      "invalid-file-title": "Neispravan tip datoteke",
+      "mpy-warning": "Ova verzija Pythonova uređivača nažalost još ne podržava dodavanje .mpy datoteka.",
+      "extension-warning": "Pythonov uređivač može učitati samo datoteke s ekstenzijama .hex ili .py."
+    },
+    "languages": {
+      "en": {
+        "title": "Engleski"
+      },
+      "es": {
+        "title": "Španjolski"
+      },
+      "pl": {
+        "title": "Poljski"
+      },
+      "hr": {
+          "title": "Hrvatski"
+      }
+    },
+    "static-strings": {
+      "buttons": {
+        "command-download": {
+          "title": "Preuzmi .hex datoteku za prijenos (flash) na micro:bit",
+          "label": "Preuzmi"
+        },
+        "command-disconnect": {
+          "title": "Odspoji s micro:bita",
+          "label": "Odspoji"
+        },
+        "command-flash": {
+          "title": "Učitaj (\"flash\") projekt izravno u micro:bit",
+          "label": "Flash"
+        },
+        "command-files": {
+          "title": "Učitaj/spremi datoteke",
+          "label": "Učitaj/spremi"
+        },
+        "command-serial": {
+          "title": "Priključi micro:bit putem serijske veze",
+          "label": "Otvori serijsku vezu",
+          "title-close": "Zatvori serijsku vezu i vrati se u uređivač",
+          "label-close": "Zatvori serijsku vezu"
+        },
+        "command-connect": {
+          "title": "Spoji se na micro:bit",
+          "label": "Poveži"
+        },
+        "command-connecting": {
+          "title": "Povezivanje s micro:bitom",
```

Comment:
@microbit-mark is it normal to translate micro:bit to `micro:bitom` or `micro:bita`?

Reply:
I think so. it looks like they usually use `micro:bitom` across the site

Reply:
Okay, thanks!